test/utils.js

```diff
@@ -0,0 +1,70 @@
+import { watchFile, readdirSync, statSync } from 'fs';
+import path from 'path';
+
+/**
+ * Ignored directories.
+ */
+
+var ignore = [ 'node_modules', '.git' ];
+
+/**
+ * Taken from: https://github.com/mochajs/mocha/blob/8a37e016e5629b46709891740c74d5f0b69929e0/lib/utils.js#L168
+ *
+ * Watch the given `files` for changes
+ * and invoke `fn(file)` on modification.
+ *
+ * @param {Array} files the array of files
+ * @param {Function} fn the function to call on modification
+ */
+exports.watch = function( files, fn ) {
+	var options = { interval: 100 };
+	files.forEach( function( file ) {
+		watchFile( file, options, ( curr, prev ) => {
+			if ( prev.mtime < curr.mtime ) {
+				fn( file );
+			}
+		} );
+	} );
+};
+
+/**
+ * Taken from: https://github.com/mochajs/mocha/blob/8a37e016e5629b46709891740c74d5f0b69929e0/lib/utils.js
+ *
+ * Ignored files.
+ *
+ * @param {string} fp file path
+ * @return {boolean} true if should ignore
+ */
+function ignored( fp ) {
+	return ! ~ignore.indexOf( fp );
```

Comment:
1. I fully agree and will switch to _.includes
2. I'm pretty curious about this.  Technically we could directly import from Mocha but that feels brittle to me.  We become tied to Mocha's implementation (undocumented functionality that they have no promise to support/keep).  Let me know what you think!